Source/src/components/ComponentImage.cpp

```diff
@@ -127,6 +125,10 @@ void Hachiko::ComponentImage::Draw(ComponentTransform2D* transform, Program* pro
     program->BindUniformFloat4("img_color", render_color->ptr());
     ModuleTexture::Bind(img_to_draw? img_to_draw->GetImageId(): 0, static_cast<int>(Hachiko::ModuleProgram::TextureSlots::DIFFUSE));
 	glDrawElements(GL_TRIANGLES, 6, GL_UNSIGNED_INT, 0);
+
+    // Unbind square & deactivate program:
+    App->ui->UnbindSuare();
```

Comment:
Shouldn't be named UnbindSquare?

Reply:
Thanks for the notice, it was not from my side, but I fixed it anyways 👍